.jenkins/dsl/jobs.groovy

```diff
@@ -37,4 +54,31 @@ void setupPrJob(String jobFolder) {
     def jobParams = getDefaultJobParams()
     jobParams.job.folder = jobFolder
     KogitoJobTemplate.createPRJob(this, jobParams)
+}
+
+
+void setupSyncJob(String jobFolder, KogitoJobType jobType) {
+    def jobParams = getJobParams('rhpam-kogito-operator-sync', jobFolder, 'Jenkinsfile.upstream-operator-sync', 'RHPAM Kogito Operator synchronizing with Kogito operator')
+    KogitoJobTemplate.createPipelineJob(this, jobParams).with {
+        parameters {
+            stringParam('DISPLAY_NAME', '', 'Setup a specific build display name')
+
+            stringParam('BUILD_BRANCH_NAME', "${GIT_BRANCH}", 'Set the Git branch to checkout')
+
+            // Build&Test information
+            stringParam('BDD_TEST_TAGS', '@rhpam', 'Execute only a subset of BDD tests')
+        }
+
+        environmentVariables {
+            env('JENKINS_EMAIL_CREDS_ID', "${JENKINS_EMAIL_CREDS_ID}")
```

Comment:
I guess that one is not useful if you don't send notifications. btw, do we want to send to Zulip or `bsig-cloud` ?

Reply:
I think we should send to Zulip.

Reply:
ok then we can let this here